airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSourceOperations.java

```diff
@@ -185,6 +185,7 @@ public void copyToJsonField(final ResultSet resultSet, final int colIndex, final
         case "path" -> putObject(json, columnName, resultSet, colIndex, PGpath.class);
         case "point" -> putObject(json, columnName, resultSet, colIndex, PGpoint.class);
         case "polygon" -> putObject(json, columnName, resultSet, colIndex, PGpolygon.class);
+        case "jsonb" -> putJsonb(json, columnName, resultSet, colIndex);
```

Comment:
Should we also take care of an array of jsonb? i.e `_jsonb`

Reply:
> Should we also take care of an array of jsonb? i.e `_jsonb`

@rodireich added support for json[] datatype
Note: 
**STANDARD SYNC** load data to the proper JsonNodes
**CDC** load data to the String. 
_A Postgres' json[] or jsonb[] column doesn't really describe any schema. So when reading such a column, Debezium converts it in string. It means that part of message is structured, part is nested as a JSON encoded string._
Currently it is a limitation of debezium and Kafka Connect API